src/swap/defi/lifi.ts

```diff
@@ -333,7 +333,7 @@ export function makeLifiPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
       spendTargets: [
         {
           memo: data,
-          nativeAmount: nativeAmount,
+          nativeAmount: mul(transactionRequest.value, '1'),
```

Comment:
Why would this ever be different from `nativeAmount`. Lifi only supports `from` quotes, so if the user specifies that they only want to send 0.1234 of an asset, we can't send more than that.